homeassistant/components/nibe_heatpump/select.py

```diff
@@ -35,11 +35,16 @@ class Select(CoilEntity, SelectEntity):
 
     def __init__(self, coordinator: Coordinator, coil: Coil) -> None:
         """Initialize entity."""
+        assert coil.mappings
         super().__init__(coordinator, coil, ENTITY_ID_FORMAT)
         self._attr_options = list(coil.mappings.values())
         self._attr_current_option = None
 
     def _async_read_coil(self, coil: Coil) -> None:
+        if not isinstance(coil.value, str):
+            self._attr_current_option = None
```

Comment:
Is this an error? Should something be logged here?

Reply:
Its an error. But we dont want to log errors all the time. Should be enough that current value is unavailable.